core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -347,4 +347,12 @@ DockerImageName getConfiguredSubstituteImage(DockerImageName original) {
         }
         return original;
     }
+
+    public String getPlatformRetry() {
+        return getEnvVarOrProperty("platform.retry", null);
```

Comment:
I think I'd like to have a bit of a think about how we name these, and how we'll document it to be easy to use.

I'm wondering if a single property containing a comma-separated list of platforms might flexible and intuitive, e.g.: `image.platforms=linux/arm64,linux/amd64`. The default value would be null in order to select the 'native' platform.

Separately, part of me still wonders whether this should be a setting that's configured at the image level - e.g. on `DockerImageName`. There are some major images that don't work when run under QEMU, so it might be better for this choice to be very deliberate and explicit, tied to the image identity.

Would be keen to hear your thoughts - I'll continue to ponder...

Reply:
Thanks for taking the time to review this. This is my first time using the library (it is great, so a huge thank you), so very conscious I have little knowledge of the patterns used or all the implications. 

The comma separated list does sound a nice approach to addressing both wanting to pin to or retry across one or more platforms, and far more elegant that two separate properties.

I considered the _DockerImageName_ but ended up going with this approach principally as my itch was to have this happen behind the scenes without adjusting a set of integration tests we have in Apache Tika that I was running within a JUnit rule quickly. It feels to me like a good place to also have this.

Reply:
@dameikle before you do anything, I should mention that we think we'd like to:

* just keep this as a configuration property (@bsideup convinced me)
* go the extra mile with respect to the default values, so that people should rarely need to touch this setting. I'd suggest leaving this out of this PR at the moment - leaving the default 'null' behaviour to be to use the native platform. But just FYI.

Reply:
> just keep this as a configuration property (@bsideup convinced me)

Errr... Have I? 😂 My thinking was that we could just fallback to `linux/amd64` if there is no image for the current arch 😅 

But yes, having this fallback configurable (with `linux/amd64` as the default value) won't hurt, as long as we have sensitive defaults that won't require a customization in 99.(9)% of cases 👍

Reply:
> Errr... Have I?

I thought so, by extension of having fallback logic (that fallback list has to be configured somewhere centrally, but it's just that we want the defaults to be automatically determined) 😂

Reply:
Ah, okay :D